analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -96,7 +96,7 @@ object VirtualEnv : CommandLineTool {
     // Allow to use versions that are known to work. Note that virtualenv bundles a version of pip. Exclude version 20.3
     // for now as is comes with a new dependency resolver that we yet need to test, see
     // http://pyfound.blogspot.com/2020/11/pip-20-3-new-resolver.html.
-    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1,20.3[")
+    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1,20.5[")
```

Comment:
Note: This should not be done without also taking the above code comment into account (i.e. test pip's new dependency resolver), and in any case it should be done in separate commit. In the context of this PR, I'd propose to simply drop this change.

Reply:
Accidentally committed this, reverted it - my local Python version is more modern version so I needed this to be able to run the functional testst